web/source/keyboards/kmwkeyboards.ts

```diff
@@ -589,9 +589,7 @@ namespace com.keyman.keyboards {
       }
 
       var Pk=keyman.core.activeKeyboard;  // I3319
-      if(Pk !== null)  // I3363 (Build 301)
-        Pk = Pk.scriptObject;
-        String.kmwEnableSupplementaryPlane(Pk && ((Pk['KS'] && (Pk['KS'] == 1)) || (Pk['KN'] == 'Hieroglyphic'))); // I3319
+      String.kmwEnableSupplementaryPlane(true);
```

Comment:
Do we need this if it is being done in keymanweb.ts during init?

Reply:
If there's no code that would ever turn it off... not in this form, no.  Could 'optimize' that out.

I'd still want to leave a comment saying "if toggling SMP support based on the active keyboard, do that here" at least.

Reply:
But I think we are removing the idea of ever turning off SMP support, no?